games-vue-client/src/components/PlayGame.vue

```diff
@@ -75,6 +77,7 @@ export default {
         this.clearActions();
         return
       }
+      this.whispered = false;
```

Comment:
The whisper resets every time the user has an action to perform. For games like Avalon when the user is having multiple actions on the same turn means that the sound will play multiple times.

Need to find a way to limit the notifications for multiple action rounds.